### PR TITLE
Add new parameter ldap_group_membership_attribute (PR#6161)

### PR DESCRIPTION
**Pull request by stonezdj**
The PR#6161 is pending on update too long, create another PR to merge it
Signed-off-by: stonezdj <stonezdj@gmail.com>

### CODE REVIEW

**Comment by JarleB** [comment]
+1

**Comment by stonezdj** [comment]
This PR is only make the group member attribute can be customized

**Comment by reasonerjt** [comment]
Given it's userscope, I assume we need to enable editing it via UI?

**Comment by JarleB** [comment]
> Given it's userscope, I assume we need to enable editing it via UI?

I think this PR now, and add ability to change via GUI with new PR later.

File: src/common/config/metadata/metadatalist.go [hunk 1]
```diff
@@ -102,6 +102,7 @@ var (
 		{Name: "ldap_uid", Scope: UserScope, Group: LdapBasicGroup, EnvKey: "LDAP_UID", DefaultValue: "cn", ItemType: &NonEmptyStringType{}, Editable: false},
 		{Name: "ldap_url", Scope: UserScope, Group: LdapBasicGroup, EnvKey: "LDAP_URL", DefaultValue: "", ItemType: &NonEmptyStringType{}, Editable: false},
 		{Name: "ldap_verify_cert", Scope: UserScope, Group: LdapBasicGroup, EnvKey: "LDAP_VERIFY_CERT", DefaultValue: "true", ItemType: &BoolType{}, Editable: false},
+		{Name: "ldap_group_membership_attribute", Scope: UserScope, Group: LdapBasicGroup, EnvKey: "LDAP_GROUP_MEMBERSHIP_ATTRIBUTE", DefaultValue: "memberof", ItemType: &StringType{}, Editable: true},
```
Comment (reasonerjt) [inline src/common/config/metadata/metadatalist.go]: Use a const?

Comment (stonezdj) [reply src/common/config/metadata/metadatalist.go]: I will update it in a batch as tech debts

**Pull request merged**